virtio-devices/src/device.rs

```diff
@@ -197,6 +197,13 @@ pub trait VirtioDevice: Send {
             offset_config.write_all(data).unwrap();
         }
     }
+
+    /// Default implementation if the device is not vhost-user or reconnection is
+    /// not supported, disconnected status will always be false for this default
+    /// one.
+    fn get_disconnected_arc(&self) -> Result<Arc<AtomicBool>, std::io::Error> {
+        Ok(Arc::new(AtomicBool::new(false)).clone())
```

Comment:
There is no need to call `clone`, is there?

Reply:
I removed the `.clone()`.